getting-started/jdbc/docker-compose-bootstrap-db.yml

```diff
@@ -0,0 +1,54 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+services:
+  polaris-purge:
```

Comment:
Talked about this offline with @singhpk234 - we should not be purging every time, there's no use of using a persistent storage if we are purging every time before we start it.

The instructions instead should make it clear that this Docker Compose file does not need to be run every time a user wants to start Polaris.

Reply:
Thanks for the pr here @adnanhemani i will incorporate the same changes for the jdbc pr